src/mrfgen/mrfgen.py

```diff
@@ -1601,7 +1601,9 @@ def create_vrt(basename, empty_tile, epsg, xmin, ymin, xmax, ymax):
         try:
             img = gdal.Open(tile)
         except RuntimeError as e:
+            errors += 1
             log_sig_exit('ERROR', 'Invalid input files', sigevent_url)
```

Comment:
This should be changed to log_sig_err so that the script doesn't exit.

Reply:
good point.. so that then causes me to re-evaluate how `img` ended up being `None`. Perhaps `gdal.Open(tile)` can return `None` without throwing an exception.  So maybe we still `log_sig_exit` to that failure and then put a try catch block around the `img.RasterCount`?

Reply:
Yeah, it's probably not throwing the exception until `img.RasterCount`.

Reply:
I'm curious then why we log and exit on the failure of `gdal.Open()` but only log an error on the `img.RasterCount`.  Any reason not to just log an error in both situations?

Reply:
I'm not sure exactly why. Maybe if `gdal.Open` failed we assumed that GDAL doesn't work at all, which would be severe enough to not bother continuing.

Reply:
ok, updated the logic.  Ready for re-review.